src/grt/grt-avhpi.adb

```diff
@@ -199,6 +272,12 @@ package body Grt.Avhpi is
       case El_Type1.Kind is
          when Ghdl_Rtik_Type_P64 =>
             if Is_Sig then
+               -- This is currently never run, but why would
+               -- this be different for generics and signals?
+               -- Maybe signals are built as structures of signals,
+               -- but generics are raw?  So signals effectively
+               -- contain lots of thin signal wrappers that we're
+               -- pointing at.
```

Comment:
Yes, generics contain the raw value, while signals are addresses to an internal structure.